ci/requirements_mxnet_virtualenv.txt

```diff
@@ -0,0 +1,6 @@
+opencv-python
```

Comment:
Are these requirements duplicated? I feel like they are pretty generic and should be in another place. could you check if there's an existing requirements.txt that we can use? otherwise, we have too many requirements files which makes us prone to miss one

Reply:
There are no suitable file with such requirements other than ci/docker/install/ubuntu_tutorials.sh

Reply:
https://github.com/apache/incubator-mxnet/blob/master/tests/requirements.txt

Reply:
We do need opencv for the tests afaik

Reply:
They are working right? Why should I change the requirements for unit tests on this PR?  it could introduce errors in CI like last time. I think that's better done in a separate PR. Is your concern addressed?

Reply:
We don't use that linked requirements.txt in our CI. Instead, we install OpenCV and all that as part of our docker setup. 

If it would introduce errors, we would see it as part of the PR validation. Please, add your requirements to the linked requirements file and delete the one you created since they are redundant. The linked file is exactly for the purpose of providing an environment to run tests, which is also the purpose of your new requirements file

Reply:
This is going to have implications on other places, in qemu we are installing those with:

81:    check_call(['sudo', 'pip3', 'install', '-r', 'mxnet/tests/requirements.txt'])

Any additional requirement can take a lot of time to compile in QEMU, I don't want to add requirements to the test files that are not needed.

I'm against the change that you propose because this reason.